loki/loki-config.yaml

```diff
@@ -0,0 +1,22 @@
+---
+server:
+  http_listen_port: 3100
+query_range:
+  deny_queries_without_filters: false
+schema_config:
+  configs:
+    - from: 2021-08-01
+      store: tsdb
+      object_store: filesystem
+      schema: v12
+      index:
+        prefix: index_
+        period: 24h
+      chunks:
+        prefix: chunk_
+        period: 24h
+      deletes:
+        retention_deletes_enabled: true
+        retention_period: 720h  
```

Comment:
Are you sure this one is correct retention (https://grafana.com/docs/loki/latest/operations/storage/retention/)?

Reply:
Nope, I am not 🤣.

Reply:
I'd try to set it for like 5 minutes or so and see if the retention actually works then :smirk: